pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1272,6 +1280,7 @@ func updatePVCStatusCapacity(pvcName string, pvc *v1.PersistentVolumeClaim, capa
 	}
 
 	pvcCopy.Status.Capacity = capacity
+	pvcCopy.Status.Conditions = []v1.PersistentVolumeClaimCondition{}
```

Comment:
This is useless?

Reply:
This is not useless. This removes conditions set on PVC after resizing has been successful and that is why it exists as separate commit as well.